Dockerfile

```diff
@@ -1,14 +1,16 @@
 #================================
 # Stage 1: Build Gitbase
 #================================
-FROM golang:1.11-alpine as gitbase-build
+FROM golang:1.11-alpine as builder
 
 ENV GITBASE_REPO=github.com/src-d/gitbase
 ENV GITBASE_PATH=$GOPATH/src/$GITBASE_REPO
 
+RUN apk add --no-cache git
+
 COPY . $GITBASE_PATH
 WORKDIR $GITBASE_PATH
-RUN go build -ldflags="-s -w" -o /bin/gitbase ./cmd/gitbase
+RUN go build -ldflags="-X main.version=$(cat version.txt || echo "undefined") -X main.build=$(date +"%m-%d-%Y_%H_%M_%S") -X main.commit=$(git log --format='%H' -n 1 | cut -c1-10) -s -w" -o /bin/gitbase ./cmd/gitbase
```

Comment:
why `git log --format='%H' -n 1 | cut -c1-10` instead of `git rev-parse --short HEAD`, like in https://github.com/src-d/ci/blob/v1/Makefile.main#L52?

Reply:
It's what was on the Makefile.main of the master branch of CI. Didn't remember the good one was v1 😄